hack/make-rules/test-cmd-util.sh

```diff
@@ -1608,7 +1608,7 @@ run_non_native_resource_tests() {
   kubectl "${kube_flags[@]}" delete foos test --cascade=true
 
   # Make sure it's gone
-  kube::test::get_object_assert foos "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::wait_object_assert foos "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
Yes, DeleteOptions overrides the strategy. I just want to know how the default strategy will affect clients other than kubectl.